README.md

```
@@ -90,6 +90,65 @@ For more information refer to [Using Docker and Docker-Compose][], this page als
 
 To configure CI for your project, run the ci-cd sub-generator (`jhipster ci-cd`), this will let you generate configuration files for a number of Continuous Integration systems. Consult the [Setting up Continuous Integration][] page for more information.
 
+## Per-Client Token Lifetime Configuration
+
+By default, all OAuth2 clients share a single global access-token and refresh-token lifetime that is configured in the
+external `uaa.yml`, for example:
+
+```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)